Maturation of getting started guide on web (#662)
MapsIndoors/docs: src/shared/getting-started/congrats.md

## Summary

You're at the end of your journey, and you've accomplished a lot! 🎉

* You learned which prerequisites is needed to start building with MapsIndoors.
* You loaded a interactive map with MapsIndoors locations and added a floor selector for navigating between floors.
* You created a search experience to search for specific locations on the map.

> * You added functionality for getting directions from one location to another.

Reviewer: ```suggestion
* You added functionality for getting directions from one Location to another.
```

> * You learned how to enable different types of live data domains in your app.

Reviewer: ```suggestion
* You learned how to enable different types of Live Data Domains in your app.
```

> This concludes the "Getting Started" tutorial. To get more inspiration on what to build next please visit our [showcase page](https://www.mapspeople.com/showcases) to see how other clients have used MapsIndoors.

Reviewer: ```suggestion
This concludes the "Getting Started" tutorial. To get more inspiration on what to build next please visit our [showcase page](https://www.mapspeople.com/showcases) to see how other clients use MapsIndoors.
```
